mlflow/spark.py

```diff
@@ -682,6 +684,8 @@ def autolog():
         # next-created MLflow run if no run is currently active
         with mlflow.start_run() as active_run:
             pandas_df = loaded_df.toPandas()
+
+        :param disable: Whether to enable or disable autologging.
```

Comment:
Can we match the param docstring of `mlflow.sklearn.autolog()` (and others) introduced by #3815 ?